### PR TITLE
Fix df from result table error message + old types 

### PR DESCRIPTION
**Pull request by mayamarom10**
fixed error message in case of none existing type 
added old type names in kusto to default dictionary #542

### CODE REVIEW

File: azure-kusto-data/azure/kusto/data/helpers.py [hunk 1]
```diff
@@ -67,16 +75,18 @@ def dataframe_from_result_table(
...
+            converter = converters_by_type.get(column_type)
         elif nullable_bools and column_type == "bool":
             converter = lambda col, df: df[col].astype(pd.BooleanDtype())
         else:
-            converter = default[column_type]
+            converter = default.get(column_type)
         if isinstance(converter, str):
             frame[column_name] = frame[column_name].astype(converter)
         else:
+            if converter is None:
```
Comment (AsafMah) [inline azure-kusto-data/azure/kusto/data/helpers.py:88]: move this before the isinstance

File: azure-kusto-data/azure/kusto/data/helpers.py [hunk 2]
```diff
@@ -18,14 +18,22 @@ def default_dict() -> Converter:
     return {
```
Comment (AsafMah) [inline azure-kusto-data/azure/kusto/data/helpers.py:18]: Add a test with an alias name, and with a missing type name

**Review by AsafMah — CHANGES_REQUESTED** [review]

File: azure-kusto-data/azure/kusto/data/helpers.py (hunk 1 above)
Comment (ohadbitt) [inline azure-kusto-data/azure/kusto/data/helpers.py:88]: Move None check before isinstance

**Review by ohadbitt — DISMISSED** [review]

File: azure-kusto-data/tests/test_helpers.py [hunk 3]
```diff
@@ -90,10 +96,21 @@ def test_dataframe_from_result_table():
...
-df_int_to_float = dataframe_from_result_table(response.primary_results[0], converters_by_type=test_int_to_float)
+ignore_missing_type = {
+    "MissingType": lambda col, frame: frame[col].astype("str"),
+}
+df_int_to_float = dataframe_from_result_table(response.primary_results[0], converters_by_type=test_int_to_float, converters_by_column_name=ignore_missing_type)
 assert type(df_int_to_float.iloc[0].RecordInt) is numpy.float64
 assert df.iloc[0].RecordInt == 5678
 
+# Testing missing type conversion
+raised = False
```
Comment (AsafMah) [inline azure-kusto-data/tests/test_helpers.py:107]: Use pytest.raises

**Review by AsafMah — APPROVED** [review]

**Pull request merged**